Directory.Packages.props

```diff
@@ -3,6 +3,7 @@
   <ItemGroup>
     <PackageVersion Include="Automapper" Version="10.1.1" />
     <PackageVersion Include="Autofac.Extras.Moq" Version="6.0.0" />
+    <PackageVersion Include="Autofac.Extensions.DependencyInjection" Version="7.1.0" />
```

Comment:
Also bring in Autofac manually (there's a newer version than this will bring in)

Reply:
added directly